cmd/config/custom-workload/custom-workload.yml

```diff
@@ -8,11 +8,19 @@ global:
         - conditionType: Ready
           metric: P99
           threshold: 15s
-indexers:
-  - esServers: ["{{.ES_SERVER}}"]
-    insecureSkipVerify: true
-    defaultIndex: {{.ES_INDEX}}
-    type: {{.INDEXING_TYPE}}
+metricsEndpoints:
+{{ if .ES_SERVER }}
+  - indexer:
+      esServers: ["{{.ES_SERVER}}"]
+      insecureSkipVerify: true
+      defaultIndex: {{.ES_INDEX}}
+      type: opensearch
+{{ end }}
+{{ if eq .LOCAL_INDEXING "true" }}
+  - indexer:
```

Comment:
Is there a reason for custom-workload to be different from all other configs where only reporting metrics are indexed locally?

Reply:
Yes, because its just a code stub for reference we have decided to exclude timeseries and reporting metrics in one of our previous PRs: https://github.com/kube-burner/kube-burner-ocp/commit/7fe178eea60a59ba12e6c2105acade2ecd08d6af